estate/models/estate_property.py

```diff
@@ -0,0 +1,116 @@
+from dateutil.relativedelta import relativedelta
+
+from odoo import _, api, fields, models
+
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.float_utils import float_compare, float_is_zero
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = 'Estate Property'
+
+    name = fields.Char(string='Title', required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(string='Available From', copy=False, default=fields.Date.today()+ relativedelta(months=3))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer(string='Living Area (sqm)')
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer(string='Garden Area (sqm)')
+    garden_orientation = fields.Selection(
+        selection=[
+            ('north','North'),
+            ('south','South'),
+            ('east','East'),
+            ('west','West'),
+        ]
+    )
+    active = fields.Boolean(default=True)
+    state = fields.Selection(
+        required=True,
+        default='new',
+        copy=False,
+        selection=[
+            ('new','New'),
+            ('offer_received','Offer Received'),
+            ('offer_accepted','Offer Accepted'),
+            ('sold','Sold'),
+            ('cancelled','Cancelled'),
+        ],
+    )
+    property_type_id = fields.Many2one(
+        comodel_name='estate.property.type',
+        string='Property Type',
+    )
+    user_id = fields.Many2one(
+        comodel_name='res.users',
```

Comment:
```suggestion
'res.users',
```
you can remove comodel_name label/attr from every relational field